### PR TITLE
Make it possible to use escalus with Erlang/OTP 21

### PR DESCRIPTION
**Pull request by michalwski**

### CODE REVIEW

**Comment by michalwski** [comment]
Here's the PR in MongooseIM esl/MongooseIM#1947

File: rebar.config [hunk 1]
```diff
@@ -3,15 +3,15 @@
             {i, [".", "include"]}
            ]}.
 
-{require_otp_vsn, "(^1[89])|^20"}.
+{require_otp_vsn, "(^1[89])|(^2[01])"}.
```
Comment (ludwikbukowski) [inline rebar.config]: if we leave support for 18 shouldn't we have regexp `(^19)|(^2[01])` now ?

Comment (michalwski) [reply rebar.config]: @ludwikbukowski I update this.

**Comment by michalwski** [comment]
Yeah, that's funny... They don't appear always, and what's more interesting they seem to be random. Previously I've seen dialyse complaining about undefined `ct:pal/3` function, now it complains about undefined `ct:get_config/1` function. 
```
escalus_ct.erl:39: Call to missing or unexported function ct:get_config/1
escalus_ct.erl:73: Call to missing or unexported function ct:get_config/1
```
It doesn't seem to be connect to changes in this PR nor to escalus code...
Any idea would be be helpful.

**Pull request merged**